scalafix-core/shared/src/main/scala/scalafix/patch/PatchOps.scala

```diff
@@ -126,4 +126,20 @@ trait PatchOps {
   def replaceSymbols(toReplace: Seq[(String, String)])(
       implicit noop: DummyImplicit,
       index: SemanticdbIndex): Patch
+
+  /**
+    * Remove whitespaces on the left side of the token. Use `includeNewLine` to
+    * indicate when newline tokens should be preserved.
+    */
+  def trimLeft(token: Token, includeNewLine: Boolean = true): Patch
```

Comment:
What do you think about having a separate method to trim only spaces? trimLeftSpaces, trimSpaces, trimRightSpaces (name up for debate). Tabs can be ignored, I'm not aware of any Scala project using tabs for indent. 

I think it could result in a cleaner user API, boolean parameters are quite awkward to both write and read.

Reply:
you are right, @olafurpg. Boolean parameters often indicate the need for two separate methods. My only concern is the explosion of methods (potentially 6 new ones) in the `RuleCtx` api (those would be inherited from `PatchOps`). If you think that's not a concern I will go for it.

Reply:
I think the eventual home for these methods will be simple defs in `object Patch`. I personally think it's OK to have more methods if it buys us a cleaner api.